src/helpers/performance-report-parser.helper.js

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs';
+import chalk from 'chalk';
+
+class PerformanceReportParse {
+  constructor() {
+    this.performanceReport = (fileName) => JSON.parse(fs.readFileSync(`reports/performance/${fileName}`, 'utf8'));
+  }
+
+  parse(fileName, maxTiming) {
```

Comment:
`parse` name usually means that the method will parse some kind of data and return it. Here it does way more - I'd call it `checkTiming` or similar.

For clarity, you could also create another method here named `parse` and put there only the part with loading and returning `{ttfb, url}` pairs. IMO this would look much better

Reply:
Changed.